evm_arithmetization/src/generation/mod.rs

```diff
@@ -80,6 +84,22 @@ pub struct GenerationInputs {
     pub block_hashes: BlockHashes,
 }
 
+pub fn tx_hash(signed_txn: &[u8]) -> anyhow::Result<H256> {
+    let hash = keccak(signed_txn);
+    Ok(hash)
+}
+
+impl GenerationInputs {
+    pub fn tx_hash(&self) -> anyhow::Result<H256> {
```

Comment:
Nit: it will ultimately not be needed, since `txn_hashes` will have its dedicated field after continuations. I think the `tx_hash` function above is sufficient.

Reply:
Agreed with Hamy, I'd rather not add something that will be removed in 1 week

Reply:
Or to refactor this function as easy getter for that field?